episodes/what-is-an-image.md

```diff
@@ -0,0 +1,540 @@
+
+## Pixels
+
+Let's start by removing all layers from the Napari viewer and opening a new 
+sample image:
+
+- Click on the top layer in the layer list and shift + click the bottom layer. 
+This should highlight all layers in blue.
+
+- Press the remove layer button ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/delete.svg
+){alt="A screenshot of Napari's delete layer button" height='30px'} 
+
+- Go to the top menu-bar of Napari and select:  
+`File > Open Sample > napari builtins > Human Mitosis`
+
+![](fig/human-mitosis-napari.png){alt="A screenshot of a 2D image of human cells 
+undergoing mitosis in Napari"}
+
+This 2D image shows the nuclei of human cells undergoing mitosis. If we zoom in 
+by scrolling, we can see that this image is actually made up of many small 
+squares with different brightness. These squares are the image's pixels (or 
+'picture elements') and are the individual units that make up all 
+digital images.
+
+If we hover over these pixels with the mouse cursor, we can see that each pixel 
+has a specific value. Try hovering over pixels in dark and bright areas of the 
+image and see how the value changes in the bottom left of the viewer:
+
+![](fig/pixel-value.png){alt="A screenshot of Napari - with the mouse cursor 
+hovering over a pixel and highlighting the corresponding pixel value"}
+
+You should see that brighter areas have higher values than darker areas (we'll 
+see exactly how these values are converted to colours in the next episode).
+
+## Images are arrays of numbers
+
+We've seen that images are made of individual units called pixels that have 
+specific values - but how is an image really represented in the computer? Let's 
+dig deeper into Napari's `Image` layers...
+
+First, open Napari's built-in Python console by pressing the console button ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/console.svg
+){alt="A screenshot of Napari's console button" height='30px'}. 
+Note this can take a few seconds to open, so give it some time:
+
+![](fig/console.png){alt="A screenshot of Napari's console"}
+
+:::::::::::::::::::::::::::::::::::::: callout
+
+### Console readability
+
+You can increase the font size in the console by clicking inside it, then 
+pressing Ctrl and + together. The font size can also be decreased with 
+Ctrl and - together.
+
+Note that you can also pop the console out into its own window by clicking the 
+small ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/pop_out.svg
+){alt="A screenshot of Napari's float panel button" height='30px'} 
+icon on the left side.
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+Let's look at the human mitosis image more closely - copy the following into 
+Napari's console:
+
+```python
+
+# Get the image data for the first layer in Napari
+image = viewer.layers[0].data
+
+# Print the image values and type
+print(image)
+print(type(image))
```

Comment:
~It may be useful to print the shape so they can see it's the same than what they see in the GUI (though with one value less)~
```suggestion
print(type(image))
print(image.shape)
```

Oh! you've got that later on! :sweat_smile: